tools/recetox_msfinder/macros.xml

```diff
@@ -0,0 +1,60 @@
+<macros>
+    <token name="@TOOL_VERSION@">0.1.0</token>
+    <xml name="requirements">
+        <requirements>
+            <container type="docker">recetox/recetox-msfinder:3.5.2-0</container>
+        </requirements>
+    </xml>
+    <xml name="creator">
+        <creator>
+            <person
+                givenName="Zargham"
+                familyName="Ahmad"
+                url="https://github.com/zargham-ahmad"
+                identifier="0000-0002-6096-224X" />
+            <organization
+                url="https://www.recetox.muni.cz/"
+                email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+                name="RECETOX MUNI" />
+        </creator>
+    </xml>
+    <xml name="input">
+        <param name="input_data" type="data" format="msp,mat">
+            <label>Input file to process</label>
+        </param>
+    </xml>
+    <xml name="parameter">
+        <param type="float" value="0.01" name="Ms2Tolerance" label="Ms2Tolerance" help="The mass tolerance for matching experimental and reference fragments." />
+        <param name="IsTmsMeoxDerivative" type="boolean" checked="False" truevalue="True" falsevalue="False" label="IsTmsMeoxDerivative"
+            help="Check TMS-MeOX derivative to use EIMS database; otherwise, MSMS database is used." />
+        <param type="integer" value="1" name="RelativeAbundanceCutOff" label="RelativeAbundanceCutOff"
+            help="The product ions more than this parameter on the basis of base peak ion are utilized for the product ion matching." />
+        <param type="float" value="0.001" name="Ms1Tolerance" label="Ms1Tolerance" help="The mass tolerance to generate formula candidates." />
+        <param type="select" name="MassToleranceType" label="MassToleranceType">
+            <option value="Da" selected="True">Da</option>
+            <option value="Ppm">Ppm</option>
```

Comment:
They could use a help text, with `ppm` standing for `parts per million` and `Da` standing for Daltons (amu).

Reply:
resolved in 50fcbda